base/src/org/compiere/model/MLandedCostAllocation.java

```diff
@@ -244,28 +247,26 @@ public int getC_DocType_ID() {
 	}
 
 	public BigDecimal getPriceActualCurrency() {
-		MInvoiceLine invoiceLine = (MInvoiceLine) getC_InvoiceLine();
-		MCurrency currency = MCurrency.get(getCtx(), getC_Currency_ID());
-		BigDecimal amount = getAmt().divide(getQty() , currency.getCostingPrecision() ,  RoundingMode.HALF_UP);
-		if (MDocType.DOCBASETYPE_APCreditMemo.equals(invoiceLine.getParent().getC_DocType().getDocBaseType()))
-			amount = amount.negate();
+		BigDecimal amount = getAmt(true, false).divide(getQty() , MathContext.DECIMAL128);

```

Comment:
Why change the rounding standard based on the currency costing rounding or schema that use all System, and was replace for a specific rounding?

Why create a crypt method , when exit getPriceActualCurrency if you need implement the logic conversion the implement into of getPriceActualCurrency not create a new method